toolchains-template-min.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF8"?>
+<!--
+  ~ Copyright 2020 The Exonum Team
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<toolchains xmlns="http://maven.apache.org/TOOLCHAINS/1.1.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/TOOLCHAINS/1.1.0 http://maven.apache.org/xsd/toolchains-1.1.0.xsd">
+  <!--
+       JDK toolchains.
+
+       `JDK_BASE_DIR` is a directory containing (symlinks to) JDK installations with their
+       directory names equal to the version (e.g., `1.8` or `11.0.2`)
+       -->
+  <toolchain>
+    <type>jdk</type>
+    <provides>
+      <version>1.8</version>
+    </provides>
+    <configuration>
+      <jdkHome>JDK_BASE_DIR/1.8</jdkHome>
```

Comment:
it would be hard to integrate it with sdkman such as it uses another format for java version distributions:

![image](https://user-images.githubusercontent.com/5055052/75141987-c54a1200-56fa-11ea-8816-9aba817688f3.png)

Reply:
It is a template for boxes that use such layout. If one doesn't — one shall enter jdkHome manually.